src/main/java/com/example/controllers/AdminController.java

```diff
@@ -0,0 +1,33 @@
+package com.example.controllers;
+
+import java.security.Principal;
+import java.util.List;
+
+import com.example.bean.Account;
+import com.example.dao.AccountDao;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping(value = "/admin")
+public class AdminController {
+
+	@Autowired
+	AccountDao accountDao;
+
+	@RequestMapping
+	public String admin(final Model model, final Principal principal) {
+
+		Account account = accountDao.findUsersByUsername(principal.getName()).get(0);
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- 2c082a0f7b35fff785c01c0a5b73088c -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - SQL Injection
_High CWE-89_
Untrusted user-supplied data is inserted into a SQL statement without adequate validation, escaping, or filtering.

A user can change the intent of the SQL query, which may inappropriately disclose or corrupt data within the database.

### How to fix
Rewrite all SQL queries constructed through dynamic concatenation to use an injection-safe query mechanism such as prepared statements with parameterized queries.

Most modern programming languages provide a feature called "parameterized queries" that allow user-supplied data to be inserted safely as values in dynamic SQL queries. Rather than construct the dynamic SQL query by concatenating user-supplied data to static SQL query string fragments, data values are identified in the query by parameter markers or variables. Dynamic data is then passed through a mechanism provided by SQL that prevents the supplied data from changing the meaning of the query.

Note: the exact syntax and use of prepared statements with parameterized queries vary from language to language.